tecken/download/views.py

```diff
@@ -240,10 +230,66 @@ def missing_symbols_csv(request):
         filename__iendswith=".sym",
     )
 
-    only = ("symbol", "debugid", "code_file", "code_id")
-    for missing in qs.only(*only):
-        writer.writerow(
-            [missing.symbol, missing.debugid, missing.code_file, missing.code_id]
-        )
+    fields = ("symbol", "debugid", "code_file", "code_id")
+    for missing in qs.values_list(*fields):
+        writer.writerow(missing)
```

Comment:
I rewrote this to use `values_list` because that skips all the work to convert the data from the db to model instances only to pull information out.